.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on: [push]
```

Comment:
Nice!!, but if we are to merge this please change this to [pull_request]

Reply:
Thx! Makes sense

Reply:
BTW: Very good CI configuration script! Works like a charm...